Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -257,24 +257,24 @@ public extension Sequence where Element: Equatable {
     ///        ["h", "e", "l", "l", "o"].contains(["l", "o"]) -> true
     ///
     /// - Parameter elements: array of elements to check.
-    /// - Returns: true if array contains all given items.
+    /// - Returns: true if sequence contains all given items.
     /// - Complexity: _O(m·n)_, where _m_ is the length of `elements` and _n_ is the length of this sequence.
     func contains(_ elements: [Element]) -> Bool {
         return elements.allSatisfy { contains($0) }
     }
 }
 
 public extension Sequence where Element: Hashable {
-    /// SwifterSwift: Check if array contains an array of elements.
+    /// SwifterSwift: Check if sequence contains elements of another sequence.
     ///
     ///        [1, 2, 3, 4, 5].contains([1, 2]) -> true
     ///        [1.2, 2.3, 4.5, 3.4, 4.5].contains([2, 6]) -> false
     ///        ["h", "e", "l", "l", "o"].contains(["l", "o"]) -> true
     ///
-    /// - Parameter elements: array of elements to check.
-    /// - Returns: true if array contains all given items.
+    /// - Parameter elements: sequence of elements to check.
+    /// - Returns: true if sequence contains all given items.
     /// - Complexity: _O(m + n)_, where _m_ is the length of `elements` and _n_ is the length of this sequence.
-    func contains(_ elements: [Element]) -> Bool {
+    func contains<T: Sequence>(_ elements: T) -> Bool where Element == T.Element {
```

Comment:
Nitpicky, but I think `S` is a better generic type name for a sequence.
```suggestion
    func contains<S: Sequence>(_ elements: S) -> Bool where Element == S.Element {
```

Reply:
Agreed, it's the standard in the Swift library